Jenkinsfile

```diff
@@ -113,24 +113,46 @@ pipeline {
             call clean_files.bat
             call run_tests.bat
             set errcode=%errorlevel%
-            rmdir "C:\\Instrument\\Apps\\EPICS"
             exit /b %errcode%
           """
           }
         }
       }
      }
+	 
+	  stage("IOC Tests") {
+        steps {
+         lock(resource: ELOCK, inversePrecedence: true) {
+           timeout(time: 1800, unit: 'MINUTES') {
+          bat """
+             if not exist "C:\\Instrument\\Apps\\EPICS\\config_env.bat" (
+                @echo ERROR Unable to find config_env.bat in linked directory
+                exit /b 1
+             )
+			 call C:\\Instrument\\Apps\\EPICS\\stop_ibex_server.bat
+             del /q C:\\Instrument\\Var\\logs\\IOCTestFramework\\*.*
+             pushd "C:\\Instrument\\Apps\\EPICS"
+			 call config_env.bat
+             make ioctests
```

Comment:
can we put the make flags to enable output here, I think jenkins will time out otherwise as there won't be any output?

Reply:
for the ioctestframework tests that take 11 hours it will only print out after its finished so jenkins might think as theres no output it has hung?

Reply:
Are you sure? From the docs at https://www.jenkins.io/doc/pipeline/steps/workflow-basic-steps/#timeout-enforce-time-limit it looks like the timeout is by default absolute. e.g. it will timeout after 1800 minutes (30 hours) regardless of output or not. You can change this to check the output but I don't thin we have